samples/deploy_and_test.py

```diff
@@ -789,27 +805,27 @@ def __str__(self):
 
 
 class CFApp:
-    def __init__(self, name, xsuaa_service_name, app_router_name=None, identity_service_name=None):
-        if name is None or xsuaa_service_name is None:
-            raise (Exception('Name and xsuua service name must be provided'))
+    def __init__(self, name, xsuaa_service_name=None, app_router_name=None, identity_service_name=None):
+        if name is None:
+            raise (Exception('Name must be provided'))
         self.name = name
         self.xsuaa_service_name = xsuaa_service_name
         self.identity_service_name = identity_service_name
         self.app_router_name = app_router_name
+        self.ias_access = None
 
     @property
     def working_dir(self):
         return './' + self.name
 
     def deploy(self):
-        logging.info("Creating Xsuaa service '{}'".format(self.xsuaa_service_name))
-        subprocess.run(
-            ['cf', 'create-service', 'xsuaa', 'application', self.xsuaa_service_name, '-c', 'xs-security.json'],
-            cwd=self.working_dir, stdout=cf_logs, check=True)
+        if self.xsuaa_service_name is not None:
+            logging.info("Creating Xsuaa service '{}'".format(self.xsuaa_service_name))
+            subprocess.run(
+                ['cf', 'create-service', 'xsuaa', 'application', self.xsuaa_service_name, '-c', 'xs-security.json'],
+                cwd=self.working_dir, stdout=cf_logs, check=True)
         if self.identity_service_name is not None:
-            logging.info("Creating IAS service '{}'".format(self.identity_service_name))
```

Comment:
this logging output should be added in `__create_ias_service()` method

Reply:
yep, done :-)